WordPress/src/main/java/org/wordpress/android/ui/uploads/UploadService.java

```diff
@@ -637,6 +631,17 @@ private PostModel updateOnePostModelWithCompletedAndFailedUploads(PostModel post
         return updatedPost;
     }
 
+    private boolean mediaBelongsToAPost(MediaModel media) {
+        PostModel postToCancel = mPostStore.getPostByLocalPostId(media.getLocalPostId());
+        if (postToCancel == null) return false;
```

Comment:
This can be simplified to `return (postToCancel != null && mUploadStore.isRegisteredPostModel(postToCancel));`

Reply:
nice! 0d5bdc5